src/test/java/net/snowflake/client/jdbc/CustomProxyLatestIT.java

```diff
@@ -148,7 +148,7 @@ public void testJVMParamsWithNonProxyHostsHonored() throws SQLException {
     System.setProperty("http.nonProxyHosts", "*");
     Connection con =
         DriverManager.getConnection(
-            "jdbc:snowflake://s3testaccoutn.us-east-1.snowflakecomputing.com", props);
+            "jdbc:snowflake://s3testaccountn.us-east-1.snowflakecomputing.com", props);
```

Comment:
This is notable.

And sigh, when I tested, I specifically picked `account` not `accountn` as the correction.